src/functions/add_a_music_album.rb

```diff
@@ -0,0 +1,13 @@
+require_relative 'add_genre'
+require_relative '../classes/music_album'
+
+def add_a_music_album
+  puts 'On spotify?[y/n]: '
+  on_spotify = gets.chomp.to_s.downcase == 'y'
+  puts 'Published date?[yyyy-mm-dd]: '
+  published_date = gets.chomp.to_s
+  music_album = MusicAlbum.new(on_spotify, published_date)
+  @music_albums << music_album
+  puts 'Music album added successfully!'
+  add_genre
+end
```

Comment:
- [ ] when i add a music album, the data is not saved instead it duplicates the last label I added watch this [video](https://www.loom.com/share/86b613e3b61c4e2689b07b752dd5ee9a?sid=2f84efa1-ebe7-484d-9af6-94df8bda5600) to understand what I mean :+1: